ui/client/components/graph/node-modal/editors/expression/BoolEditor.tsx

```diff
@@ -1,17 +1,23 @@
 import i18next from "i18next"
 import {isEmpty} from "lodash"
-import React, {ReactNode} from "react"
-import ExpressionWithFixedValues from "./FixedValuesEditor"
+import React from "react"
+import FixedValuesEditor from "./FixedValuesEditor";
+
+type BoolEditor<P> = React.FunctionComponent<P> & {
+  switchableTo: Function
+  switchableToHint: string
+  notSwitchableToHint: string
+}
 
 type Props = {
-  expressionObj: {};
-  onValueChange: Function;
-  readOnly: boolean;
-  className: string;
+  expressionObj: any
```

Comment:
avoid any, use $TodoType to distinguish (yet) unknown types from truly any type

Reply:
Done